### PR TITLE
Remove unnecessary components + rename things to Bizyhood - BW-561

### PR DESCRIPTION
**Pull request by dgmdan**
Resolves BW-561

### CODE REVIEW

File: Bizyhood/Ajax.php [hunk 1]
```diff
@@ -1,102 +1,30 @@
 <?php
 /**
  * This file contains a class which provides the AJAX callback functions required
- *  for Broadstreet.
+ *  for Bizyhood.
  *
- * @author Broadstreet Ads <labs@broadstreetads.com>
+ * @author Bizyhood Ads <labs@bizyhoodads.com>
```
Comment (eugenebiz) [inline Bizyhood/Ajax.php]: @author Bizyhood, Inc <support@bizyhood.com>

File: Bizyhood/Core.php [hunk 2]
```diff
@@ -138,255 +85,89 @@ static function install()
...
 
         # -- Below is core functionality --
         add_action('admin_menu', 	array($this, 'adminCallback'     ));
         add_action('admin_init', 	array($this, 'adminInitCallback' ));
-        add_action('init',          array($this, 'addZoneTag' ));
-        add_action('init',          array($this, 'businessIndexSidebar' ));
-        add_action('admin_notices',     array($this, 'adminWarningCallback'));
-        add_action('widgets_init', array($this, 'registerWidget'));
-        add_shortcode('broadstreet', array($this, 'shortcode'));
+        // add_action('init',          array($this, 'addZoneTag' ));
```
Comment (eugenebiz) [inline Bizyhood/Core.php]: Do we need to keep any of the commented code?

File: Bizyhood/Model.php [hunk 3]
```diff
@@ -2,14 +2,14 @@
 /* 
  * 
  *
- * @author Broadstreet Ads <labs@broadstreetads.com>
+ * @author Bizyhood Ads <labs@bizyhoodads.com>
```
Comment (eugenebiz) [inline Bizyhood/Model.php]: Ditto for all author tags.

File: Bizyhood/Core.php (hunk 2 above)
Comment (dgmdan) [inline Bizyhood/Core.php]: Removed.

File: Bizyhood/Model.php (hunk 3 above)
Comment (dgmdan) [inline Bizyhood/Model.php]: Removed.

File: Bizyhood/Core.php [hunk 4]
```diff
@@ -138,255 +83,63 @@ static function install()
...
-    
-
     /**
-     * A callback executed whenever the user tried to access the Broadstreet admin page
+     * A callback executed whenever the user tried to access the Bizyhood admin page
      */
     public function adminCallback()
     {
-        $icon_url = 'http://broadstreet-common.s3.amazonaws.com/broadstreet-blargo/broadstreet-icon.png';
+        $icon_url = 'http://bizyhood-common.s3.amazonaws.com/bizyhood-blargo/bizyhood-icon.png';
```
Comment (eugenebiz) [inline Bizyhood/Core.php]: This url doesn't exist.

File: Bizyhood/Utility.php [hunk 5]
```diff
@@ -2,41 +2,20 @@
...
- *  parts of Broadstreet
+ *  parts of Bizyhood
  */
-class Broadstreet_Utility
+class Bizyhood_Utility
 {
-    const KEY_ZONE_CACHE = 'BROADSTREET_ZONE_CACHE';
-    const KEY_RW_FLUSH   = 'BROADSTREET_RW_FLUSH';
-    const KEY_NET_INFO   = 'BROADSTREET_NET_INFO';
+    const KEY_ZONE_CACHE = 'BIZYHOOD_ZONE_CACHE';
```
Comment (eugenebiz) [inline Bizyhood/Utility.php]: It looks like you're doing some find/replace action. Are these variables still going to function correctly?

File: Bizyhood/Views/listings/index.php [hunk 6]
```diff
@@ -1,20 +1,20 @@
 <style>
```
Comment (eugenebiz) [inline Bizyhood/Views/listings/index.php]: Can we externalize these and any other defined styles into a stylesheet?

Comment (eugenebiz) [inline Bizyhood/Views/listings/index.php]: Ignore the stylesheet request.

**Comment by eugenebiz** [comment]
Ship it.

File: Bizyhood/Core.php (hunk 4 above)
Comment (dgmdan) [inline Bizyhood/Core.php]: Fixed

File: Bizyhood/Utility.php (hunk 5 above)
Comment (dgmdan) [inline Bizyhood/Utility.php]: Yes, this won't effect anything.

**Pull request merged**